music-player/src/components/AddSongToPlaylist.tsx

```diff
@@ -0,0 +1,80 @@
+import { Modal, Button } from 'antd';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+
+import { State } from '../types';
+import { getPlaylists } from '../reducers';
+import { getAddSongToPlaylistAction } from '../actions';
+
+class AddSongToPlaylist extends React.Component<
+  {
+    playlistsNotIncludeSongIndex: string[];
+    realIndex: number;
```

Comment:
`realIndex` is a weird name.
What is an unreal index? is there another index here to be confused with?